util/build_compilation_database.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -ex
+
+BASE_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}/" )/.." &> /dev/null && pwd )
+
+TMP_DIR=`mktemp -d`
+echo ${TMP_DIR}
+AWS_LC_BUILD="${TMP_DIR}/AWS-LC-BUILD"
+
+MY_CMAKE_FLAGS=("-GNinja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON")
```

Comment:
looks like this export option was added in CMake 2.8.5. do we care about versions older than that? if so, suggest checking CMake version and exiting w/ descriptive error, else this option will be ignored and the script will silently fail to produce the compilation DB.

Reply:
AWS-LC won't build with CMake prior to 3.2.  We've got CI to help ensure we don't regress on that here: https://github.com/aws/aws-lc/blob/main/.github/workflows/cmake.yml